Doc/library/csv.rst

```diff
@@ -75,7 +75,7 @@ The :mod:`csv` module defines the following functions:
    A short usage example::
 
       >>> import csv
-      >>> with open('eggs.csv', newline='') as csvfile:
+      >>> with open('eggs.csv', newline=None) as csvfile:
```

Comment:
As `newline` is already set to `None` by default, I think there should be just `with open('eggs.csv') as csvfile:`.